README.md

```diff
@@ -52,3 +56,27 @@ issue will be addressed in a future version.
 You can become root with `$ sudo ...` without needing a password. If you ever
 really need passwords for some accounts, note that the default Vagrant settings
 are still working, i.e. `root:vagrant` and `vagrant:vagrant`.
+
+## Mail server
+
+You do not need to change anything if you like to send emails via the
+self-administration. But if you like to read the sent mails, you need to use
+telnet. This is useful if you like to register a user and want to activate him
```

Comment:
Nevertheless, let's document the telnet stuff anyway, because it's the fastest and easiest way. But then you should also install telnet :)